### DISCUSSION TITLE
Tools

### DISCUSSION THREAD

**Discussion by IanChamberlain** [datamesh-architecture/datamesh-architecture.com#39] [General]
How do new tools such as Microsoft's Fabric fit a data mesh architecture?

**Comment by jochenchrist** [comment]
They fit great, as they optimize for the data user experience.

In Data Mesh, your goal as data product team is to provide your domain data in a way that is easiest and trustworth for your data consumers, or simply use the data for your own analysis.

With Microsoft Fabric, your users use data analytics tools they are used to work with: MS Excel, PowerBI, Copilot, and powerful Notebooks while having all data products available.
